packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.test.tsx

```diff
@@ -137,8 +137,12 @@ describe('FocusTrapZone', () => {
       }
     });
 
-    // Focus the first button.
-    ReactTestUtils.Simulate.focus(buttonA);
```

Comment:
Lets still make sure that focusing the first first element first works, keep this in, test it, and then focus last element and make sure it wraps correctly.

Reply:
I may split this into it's own test in that case. The reason being that the issue does not reproduce if you shift+tab from the first element to wrap backwards before trying to flow out of the end.